### ISSUE TITLE
Testing VOT2016 Evaluation yields worse results

### ISSUE THREAD

**Issue by lukaswals** [martin-danelljan/ECO#44]
Hello Martin, first I wanted to thank you for an amazing job you have made alongside your peers.

As the title said, I'm getting different results compared to what you showed in the ECO paper.
My first tests were done using the ECO version available at the VOT Challenge website, and obtained the following results:

EAO Values 1st try
![baseline_expected_overlap_expected_overlaps_baseline_all](https://user-images.githubusercontent.com/32473858/46189351-311f2400-c329-11e8-9d33-45dce8c17b68.png)

![EAO and Sequence Length](https://user-images.githubusercontent.com/32473858/46189494-ca4e3a80-c329-11e8-9530-de35ea5bee8f.png)

![EAO Table](https://user-images.githubusercontent.com/32473858/46189651-96bfe000-c32a-11e8-9fb0-396893bf0f29.png)

After seeing those results, I decided to check the code in this GitHub, and found there was a minor difference with the VOT version.
In the code downloaded from VOT the "**learning_rate**" param was set to "**0.011**", and in this repo it is set to "**0.012**". The other difference was the presence of a "**neglect_higher_frequency**" parameter that is not present in the code from this repo.
After applying those changes, I run the evaluation and analysis again and obtained the following results:

EAO Values 2nd try

![image](https://user-images.githubusercontent.com/32473858/46189619-6d9f4f80-c32a-11e8-8281-384100e9f19e.png)

![image](https://user-images.githubusercontent.com/32473858/46189625-755ef400-c32a-11e8-9ec8-36a21add6859.png)

![image](https://user-images.githubusercontent.com/32473858/46189640-8871c400-c32a-11e8-8937-78e055b9a77a.png)

I still couldn't reproduce the same results. Why do you think this can be happening and end up with a 0.04 EAO difference?

For the test I used the followings 

- Windows 10 platform
- VOT Toolkit v6.0.3
- Matlab R2017a
- MatConvNet 1.0-beta24

Again, thanks for the hard work.
Best regards, Lucas.

**Comment by martin-danelljan** [comment]
Hi Lucas.

I am not sure what is the problem exactly. I guess you are using the VOT2016_DEEP_settings.m runfile? Are you running on CPU or GPU?

The VOT toolkit is constantly updating, and they are changing how the evaluation measures are computed from time to time. This is the most likely cause I would say. From my own experience, VOT results are pretty challenging to reproduce for any tracker due to this, and the added randomness in the VOT metrics which are mainly caused by the restart criteria. Other than that, differences in OS and Matlab version can also affect the performance. There is also a risk that I accidentally uploaded the wrong parameter file (it was a while ago now).

**Comment by lukaswals** [comment]
Thank you for your answer Martin.

As you said, I used the settings from 'VOT2016_DEEP' file and run the tracker on GPU. Also I used the following platform/software+versions :

- Windows 10 
- VOT Toolkit v6.0.3 (downloaded from this website https://bintray.com/votchallenge/toolkit/current )
- Matlab R2017a
- MatConvNet 1.0-beta24

Probably these differences between the tools you used and the ones I use can be the cause that I cannot reproduce the results, right?

Best regards,
Lucas.
